### PR TITLE
[Windows] WebGL support added

### PR DESCRIPTION
**Pull request by tero-paananen**
# Summary

WebGL support added. WebView can show now WebGL content. 

Read more from https://docs.microsoft.com/en-us/microsoft-edge/webvr/webvr-in-webview

See the updated example app:

webgl

## Compatibility

Windows
**React Native Windows 0.62.x**

## Checklist

- [x] I have tested this on a device and a simulator

### CODE REVIEW

**Comment by tero-paananen** [comment]
Fixes issue #1621

**Comment by tero-paananen** [comment]
Test release build fails in command `npx react-native run-windows --root example --arch x64 --release --no-packager --no-deploy --logging`

i think that there might be missing attribute `--use-react-native-windows` ?

**Comment by tero-paananen** [comment]
@jamonholmgren is WebGL support needed? I need it for my app...

**Comment by tero-paananen** [comment]
@Titozzz  is WebGL support needed? I need it for my app...

**Comment by Titozzz** [comment]
I'll forward the issues to the windows team

**Comment by tero-paananen** [comment]
> I'll forward the issues to the windows team

Thanks!  My another PR is also very important, could you forward this also https://github.com/react-native-webview/react-native-webview/pull/1617

File: windows/ReactNativeWebView/ReactWebView.cpp
Comment (NickGerleman) [inline windows/ReactNativeWebView/ReactWebView.cpp:142]: nit: typo

**Review by NickGerleman — APPROVED** [review]

File: windows/ReactNativeWebView/ReactWebView.cpp
Comment (asklar) [inline windows/ReactNativeWebView/ReactWebView.cpp:147]: "When you allow the requested permission, subsequent calls to Allow, Deny, or Defer the permission request result in an error."
(same with Deny)

Does that mean you have to cache the last result so that you don't make a new call to Allow/Deny which can result in an exception being thrown?

File: windows/ReactNativeWebView/ReactWebView.cpp
Comment (asklar) [inline windows/ReactNativeWebView/ReactWebView.cpp:149]: It's possible there could be other pieces of the webview that want to register for webview permission request callbacks (even though that's not the case today), I wonder if it would be better to not deny these and let other listeners deal with it; this might result in some permission requests being raised more than once but would give the control more flexibility to decentralize access.

File: windows/ReactNativeWebView/ReactWebView.cpp
Comment (tero-paananen) [reply windows/ReactNativeWebView/ReactWebView.cpp:147]: I does not have any problem with this implementation. All examples in the internet what i have found are implemented  as quite same. Like this https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/web-view

File: windows/ReactNativeWebView/ReactWebView.cpp
Comment (tero-paananen) [reply windows/ReactNativeWebView/ReactWebView.cpp]: 0485c96fdb1b3d4a8d742df054eaddb14283eeff

File: windows/ReactNativeWebView/ReactWebView.cpp
Comment (tero-paananen) [reply windows/ReactNativeWebView/ReactWebView.cpp]: a5933a87a3d638bca6c4d49321d12b6755fe69ad

**Comment by tero-paananen** [comment]
@NickGerleman  and @asklar this PR is ready from my side now. I will start to fix my another PR about messaging now...

**Comment by tero-paananen** [comment]
I will fix the conficts soon

**Comment by tero-paananen** [comment]
@Titozzz @NickGerleman  @asklar  this PR is also ready for merge. I just fixed conficts with master (my messaging impl)

**Comment by tero-paananen** [comment]
Git hub action bot did close my issue for this PR. Does this mean that i have to remove this PR?

**Comment by asklar** [comment]
> Git hub action bot did close my issue for this PR. Does this mean that i have to remove this PR?

I think @jamonholmgren can resurrect it if needed. looks like the tests were failing?

**Comment by zwang** [comment]
Hi guys, any plan to merge this? The failed Windows CI seems to be not related to this. It seems to be failing for every Pull requests.  Maybe should not block this pull requests. 

Thanks for the great work.

**Comment by tero-paananen** [comment]
@asklar  Do i have any change to get this PR merged?

**Comment by tero-paananen** [comment]
If i have any change i may once again get pull from master and test before merge

**Pull request closed without merging**
